tests/PHPUnit/Unit/DbHelperTest.php

```diff
@@ -74,6 +74,7 @@ public function getTestQueries()
             ['SELECT  /*+ MAX_EXECUTION_TIME(1500) */  * FROM table', 'SELECT * FROM table', 1.5],
             ['SELECT  /*+ MAX_EXECUTION_TIME(20000) */  column FROM (SELECT * FROM table)', 'SELECT column FROM (SELECT * FROM table)', 20],
             ['SELECT * FROM table', 'SELECT * FROM table', 0],
+            ['SELECT /*+ MAX_EXECUTION_TIME(1000) */ * FROM table', 'SELECT /*+ MAX_EXECUTION_TIME(1000) */ * FROM table', 3.5], //Should not append MAX_EXECUTION_TIME as 3500 if already present
```

Comment:
Where does the 3.5 come from?

Reply:
@michalkleiner Its there to test the behaviour to not append `/*+ MAX_EXECUTION_TIME(3500) */`, Any plugin or method sets a `MAX_EXECUTION_TIME` and we have 3.5 seconds set as `archiving_query_max_execution_time:3500` config setting but it should not append and rather use the value set by plugins or any other method

Reply:
Right, ok, thanks. Now I see the number actually doesn't matter in this particular test, can be anything.

Reply:
```suggestion
            ['SELECT /*+ MAX_EXECUTION_TIME(1000) */ * FROM table', 'SELECT /*+ MAX_EXECUTION_TIME(1000) */ * FROM table', 3.5], // should not append/change MAX_EXECUTION_TIME if already present
```